packages/expo/src/useOAuth.ts

```diff
@@ -1,26 +1,35 @@
 import { useSignIn, useSignUp } from '@clerk/clerk-react';
-import type { OAuthStrategy } from '@clerk/types';
+import type { OAuthStrategy, SetActive, SignInResource, SignUpResource } from '@clerk/types';
 import * as AuthSession from 'expo-auth-session';
 import * as WebBrowser from 'expo-web-browser';
 
 export type UseOAuthFlowParams = {
   strategy: OAuthStrategy;
   redirectUrl?: string;
+  unsafeMetadata?: SignUpUnsafeMetadata;
+};
+
+export type StartOAuthFlowReturnType = {
+  createdSessionId: string;
+  setActive?: SetActive;
+  signIn?: SignInResource;
+  signUp?: SignUpResource;
```

Comment:
❓ aren't `signIn` and `signUp` always available in response? should we omit `?` ?

Reply:
They are not always available, especially until ClerkJS loads.